tests/scripts/all.sh

```diff
@@ -2527,9 +2533,7 @@ component_test_psa_crypto_config_accel_all_curves_except_p192 () {
     # Explicitly disable all SW implementation for elliptic curves and enable
     # their accelerated version (this excludes the built-in automatic inclusion
     # from "config_psa.h")
-    # Just leave SW implementation for the curve with the smallest bit size
-    # (MBEDTLS_ECP_DP_SECP192R1_ENABLED) for allowing to build with ECP_C.
-    # scripts/config.py unset MBEDTLS_ECP_DP_SECP192R1_ENABLED
+    scripts/config.py unset MBEDTLS_ECP_DP_SECP192R1_ENABLED
```

Comment:
Minor: the compete enumeration is a bit tedious, perhaps we should use a for loop as done in `component_test_everest_curve25519_only()`?

Perhaps the same trick can be done with `ECC_xxx` symbols by sed-ing `include/psa/crypto_config.h` while at it.

Reply:
> Perhaps the same trick can be done with ECC_xxx symbols by sed-ing include/psa/crypto_config.h while at it.

Ok for the 1st part, but this one is not clear to me. Don't we want `PSA_WANT_ECC_` symbols when building the library? I thought that `PSA_WANT_ECC_` + `MBEDTLS_PSA_ACCEL_` combo on the main library side is what we wanted. Am I wrong?

Reply:
Yes, I meant just to get the list of symbols, then add them to `loc_accel_list` - not unset them with `scripts/config.py` indeed! Something like:
```
for CURVE in $( sed ... ); do
   loc_accel_list="$loc_accel_list $CURVE"
done
```

Reply:
Ok, I see, I misunderstood the previous comment then. I got it now ;)

Reply:
Actually now that I think of it, it's not just cosmetic, getting the list from the config file would also be more future-proof as we might add new curves, there are at least two Edwards curves that are serious candidates if we can ever get to it...